scripts/setup-helper-functions.sh

```diff
@@ -177,34 +183,37 @@ function wget_and_untar {
   local URL=$1
   local DIR=$2
   mkdir -p "${DEPENDENCY_DIR}"
-  pushd "${DEPENDENCY_DIR}"
+  pushd "${DEPENDENCY_DIR}" || exit
```

Comment:
Did shellcheck catch this?

Reply:
Yes its [SC2164](https://github.com/koalaman/shellcheck/wiki/SC2164).